INFO: Started
INFO: SessionName = 28-07-2023--03-10-benchmark_2__multiple_pokemon_0_00___mul_0_00_rapidash_alakazam_machamp_golem_arcanine-txt
INFO: ('logs/', 'folder already exists.')
INFO: Runtime #1371# create_log_folder: TotalSec: 0.000 == 00.000 seconds, 00 minutes, 00 hours, 00 days
INFO: -Argparser-
DEBUG: argParser.fileName benchmark_2/multiple_pokemon_0_00/_mul_0_00_rapidash_alakazam_machamp_golem_arcanine.txt
DEBUG: argParser.timeLimit 7200
DEBUG: argParser.model Model.Standard
INFO: Runtime #1372# argparser: TotalSec: 0.001 == 00.001 seconds, 00 minutes, 00 hours, 00 days
INFO: -FileParser-
DEBUG: > File exists: benchmark_2/multiple_pokemon_0_00/_mul_0_00_rapidash_alakazam_machamp_golem_arcanine.txt
DEBUG: your current path to the project is : 
 /home/rzepka/runColor2/SatHelper/examples
DEBUG: your folder contains these files: 
['script.sh', 'benchmark_hard_81.txt', 'glucose', 'quicksand', 'Archive.zip', 'main3.py', 'nohup_5Mail.out', 'simple.cnf', 'kissat_mac', '__pycache__', 'benchmark2.txt', 'runExperiment2.sh', 'glucose-4.2.1', '.DS_Store', 'benchmark18-01-2023--11-17-24.txt', '04-04-2023--23-33-benchmark-txt', 'benchmark_easy_1.txt', '_nohup_process_gif.out', 'mainTest.py', 'example.txt', '_nohup_process.out', 'benchmark5.txt', 'gui_11_old_commented.py', 'benchmark21-01-2023--10-54.txt', '_could_not_run.txt', 'abra.txt', '_nohup_process_gif_new.out', 'email_credentials-2.txt', 'runExperiment.sh', 'benchmark18-01-2023--11-17-27.txt', 'benchmark18-01-2023--11-17-26.txt', 'glucose-folder', 'benchmark21.txt', 'kissat_linux', 'benchmark3.txt', 'processed_files.txt', 'coloring.cnf', 'benchmark18-01-2023--11-17.txt', 'benchmark_hard_71.txt', 'runTests.sh', '.idea', 'benchmark_easy_15.txt', '_original examples', 'output.txt', 'runTests-results.txt', 'nohup.out', 'benchmark4.txt', 'sample3.cnf', 'benchmark.txt', 'benchmark6.txt', 'email_credentials-1.txt', 'src', '_running_process.txt', 'solver', 'sent_emails.txt', 'benchmark1.txt', 'tests-scripts', 'glucose_mac', 'pyproject.toml', 'email_credentials3.txt', 'runExperiment3.sh', 'graph_coloring.cnf', 'special-heptomino2.txt', 'email_credentials-3.txt', 'benchmark_2', 'benchmark18-01-2023--11-17-1.txt', 'results_os_walk.txt', 'main5Mails.py', 'tests', 'runTestsAndSaveResults.sh', '.pytest_cache', 'img', '_nohup_process_rC2.out', 'simple2.cnf', '_main_check_process_rC2.py', 'benchmark18-01-2023--11-17-5.txt', 'results_find.txt', 'runExperimentAll.sh', 'benchmark18-01-2023--11-17-9.txt', 'sleep.py', 'email_data.txt', 'logs', 'data']
ERROR: Error not all the same: [29, 29, 29, 29, 29, 29, 29, 28, 29, 29, 29, 29, 29, 29, 29, 29, 29, 28, 29, 29, 29, 29, 29, 29, 29, 29, 29]
INFO: Error occurred in <function FileParser._check_all at 0x7f086f5ce940> in Profiler: Error not all the same: [29, 29, 29, 29, 29, 29, 29, 28, 29, 29, 29, 29, 29, 29, 29, 29, 29, 28, 29, 29, 29, 29, 29, 29, 29, 29, 29]
INFO: Runtime #1373# fileparser: TotalSec: 0.011 == 00.011 seconds, 00 minutes, 00 hours, 00 days
INFO: Runtime #1369# main without data saving: TotalSec: 0.015 == 00.015 seconds, 00 minutes, 00 hours, 00 days
INFO: Runtime #1370# complete_main: TotalSec: 0.025 == 00.025 seconds, 00 minutes, 00 hours, 00 days
